tools/smoke_tests.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""
+This script tests end-to-end functionality using the Civis Python client.
+It uses the live Civis API and Redshift, so a valid CIVIS_API_KEY is needed.
+
+This is based on a similar script for the R client:
+https://github.com/civisanalytics/civis-r/blob/master/tools/integration_tests/smoke_test.R
+"""
+
+import io
+import logging
+import time
+
+import civis
+import pandas as pd
+from sklearn.datasets import load_iris
+
+
+def main():
+    logging.basicConfig(format="", level=logging.INFO)
+
+    logger = logging.getLogger("civis")
+
+    t0 = time.time()
+
+    database = "redshift-general"
+    client = civis.APIClient(resources="all")
+
+    # Test read_civis and read_civis_sql produce the same results.
+    logger.info('Testing reading from redshift...')
+    sql = "SELECT * FROM datascience.iris"
+    df1 = civis.io.read_civis_sql(
+        sql=sql, database=database, use_pandas=True, client=client)
+    df2 = civis.io.read_civis(
+        table="datascience.iris", database=database, use_pandas=True,
+        client=client
+    )
+    assert df1.shape == (150, 6)
+    pd.testing.assert_frame_equal(df1, df2)
```

Comment:
is it possible the ordering to the two tables comes back differently from the database and this results in `false`?

Reply:
oh, good point.  i haven't seen that, but i'll add an order by or sort after downloading

Reply:
@patr1ckm, the r client test does the same thing [here](https://github.com/civisanalytics/civis-r/blob/master/tools/integration_tests/smoke_test.R#L29), right?